concent_api/conductor/models.py

```diff
@@ -49,18 +55,18 @@ class BlenderSubtaskDefinition(Model):
     For each VerificationRequest there must be exactly one BlenderSubtaskDefinition in the database.
     """
 
-    class OutputFormat(ChoiceEnum):
-        JPG = 'jpg'
-        PNG = 'png'
-        EXR = 'exr'
+    class OutputFormat(IntEnum):
+        JPG = 1
+        PNG = 2
+        EXR = 3
 
     # Foreign key to VerificationRequest. Can't be NULL and must be unique.
     verification_request = OneToOneField(VerificationRequest, unique=True, related_name='blender_subtask_definition')
 
     # Type of the output image to be produced by Blender. This determines the file extensions.
     # Only formats supported by Blender should be allowed here.
     # This value is passed to Blender using the -F command-line option.
-    output_format = CharField(max_length=32, choices=OutputFormat.choices())
+    output_format = IntegerField(choices=choiceadapter(OutputFormat))
```

Comment:
Could `choiceadapter` be done automatically on instantiating a field ? How about making a subclass of `IntegerField` and adding this function to overwritten method which prepares choices ?

Reply:
done

Reply:
done